code/modules/vehicles/interior/interactable/seats.dm

```diff
@@ -108,7 +108,7 @@
 				to_chat(user, SPAN_WARNING("You are unable to use heavy weaponry."))
 			return
 
-		if(!HAS_TRAIT(user, TRAIT_OPPOSABLE_THUMBS))
+		if(!user.IsAdvancedToolUser())
```

Comment:
Though this likely fixes the problem, you likely want to test both, or just give humans the trait (likely better if there's still other places that PR caused problems). The source of it is because of this toggle that's been a reoccurring april fools PR that was decided to just be incorporated: https://github.com/cmss13-devs/cmss13/pull/6034/files#diff-5d524b4f8747a9837378e330b2ec7dc3be0b3866c42591d0f84d735f4aec52f7

Reply:
Reverted that logic change and granted all humanoid species `TRAIT_OPPOSABLE_THUMBS`.